feature/google/src/test/java/BillingWrapperTest.kt

```diff
@@ -653,37 +615,6 @@ class BillingWrapperTest {
         }
     }
 
-    @Test
-    fun `getting all purchases gets both subs and inapps`() {
```

Comment:
any test completely deleted from this common file should now be in both flavor-specific files, with the unique implementation there